src/target/at32f43x.c

```diff
@@ -289,6 +296,24 @@ static bool at32f43_detect(target_s *target, const uint16_t part_id)
 	return true;
 }
 
+static bool at32f405_detect(target_s *target, const uint16_t part_id)
+{
+	DEBUG_TARGET("%s: part_id %03X\n", __func__, part_id);
```

Comment:
This looks like it would be better suited going in the probe routine after all the read-out of IDs is done but before the first dispatch to a detect routine - probably have it print out both `idcode` and `project_id`?

Reply:
Moved to common probe.